### ISSUE TITLE
Extend support for chapter keyboard shortcuts

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/almanac.httparchive.org#1637]
There are some little-known keyboard shortcuts that work on chapter pages:

- `<left arrow>` or `<p>` keys: navigate to the previous chapter
- `<right arrow>` or `<n>` keys: navigate to the next chapter

What other shortcuts might be useful, especially from an accessibility perspective? Some ideas:

- skip to main content
- toggle show/hide all figure descriptions
- next/previous section
- if #1617 becomes a thing, maybe a shortcut for each CTA
- `<?>` key to reveal a UI listing available keyboard shortcuts

How should these shortcuts change for localized pages: Is it best practice to swap left/right on RTL pages? Do "n" and "p" lose their meaning in non-English languages?

**Comment by bazzadp** [comment]
Been researching this and not 100% sure this is a good idea as might cause more accessibility issues than it solves.

I already had to remove the arrow key changes as they triggered a page change too often (e.g. scrolling on an overflow table or code block). I’d have similar concerns about too many other short cut keys in case they overlap existing keyboard shortcuts people might have or are accidentally triggered. It seems there’s no useful advice out there in this, except warning about shortcut keys.

Screen readers already have short cuts to navigate about the page, and for most of the rest of the users it’s hard to beat the tried and trusted scroll, page up/down, find and the index table of contents we have.

I think it’s worth keeping the `p`/`n` (and the `<`/`,` and `>`/`.` equivalents respectively) since they are useful for power users and not heard any complaints but I’m kind of hesitant to add to this.

Saying that I do think we should have something to expand all descriptions. If only for reviewing and spell checking! Not show if we want a short cut key or a query param as honestly not sure it’s that useful for everyone else.

**Comment by rviscomi** [comment]
@bazzadp should we close this?

**Comment by bazzadp** [comment]
Might be still good to add a `?` dialog for the existing short cuts?

**Comment by shantsis** [comment]
Single character keys are problematic for accessibility. [According to WCAG](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html):

> **Success Criterion 2.1.4 Character Key Shortcuts (Level A)**: If a keyboard shortcut is implemented in content using only letter (including upper- and lower-case letters), punctuation, number, or symbol characters, then at least one of the following is true:
> 
> Turn off
> - A mechanism is available to turn the shortcut off;
> Remap
> - A mechanism is available to remap the shortcut to include one or more non-printable keyboard keys (e.g., Ctrl, Alt);
> Active only on focus
> - The keyboard shortcut for a user interface component is only active when that component has focus.
